build.gradle

```diff
@@ -18,7 +18,7 @@ buildscript {
 
 plugins {
     id("base")
-    id("org.jetbrains.kotlinx.binary-compatibility-validator") version "0.15.0-Beta.1"
```

Comment:
It'd be convenient if the `kotlin-community/dev` branch was updated to include the recent changes from master, because the BCV version can be defined in a single place:

https://github.com/Kotlin/kotlinx-benchmark/blob/4f0dffb59eae072c1c263496ec15bff32811599d/gradle/libs.versions.toml#L4

Would that be possible?

Reply:
> Would that be possible?

Well, it's not only possible but also mandatory, I guess. :)